.github/workflows/publish_pypi.yml

```
@@ -102,9 +102,9 @@ jobs:
       - name: Build wheels on Linux and MacOS
         run: python -m cibuildwheel --output-dir wheelhouse
         env:
-          # TODO: openblas no longer available on centos 7 i686 image, use blas instead for now
+          CIBW_ARCHS_LINUX: x86_64
           CIBW_BEFORE_ALL_LINUX: >
-            yum -y install blas-devel lapack-devel &&
+            yum -y install openblas-devel lapack-devel &&
             bash build_manylinux_wheels/install_sundials.sh 5.8.1 6.5.0
```

SuiteSparse is outdated in our workflows too. Could you build wheels after making this change and see if everything works on Linux?
```suggestion
            bash build_manylinux_wheels/install_sundials.sh 6.0.3 6.5.0
```